osmosis-1/osmosis.zone_assets.json

```diff
@@ -3428,8 +3428,8 @@
         "liquid_staking",
         "sail_initiative"
       ],
-      "osmosis_verified": false,
-      "osmosis_unlisted": true,
+      "osmosis_verified": true,
+      "osmosis_unlisted": false,
```

Comment:
We can remove the osmosis_unlisted property entirely. Actually, could you please replace it with listing date?

Reply:
yes ill make this update

Reply:
Bumping this. We seem to have stopped right before the finish line.

Reply:
added listing timestamp and removed the `osmosis_unlisted` property